.buildkite/pipeline.yml

```diff
@@ -697,3 +697,57 @@ steps:
     trigger: "beats-agent-serverless-tests"
     build:
       branch: "main"
+
+  - label: "Trigger DRA (SNAPSHOT)"
+    if: build.pull_request.id != null
+    plugins:
+      - monorepo-diff#v1.0.1:
+          diff: "git diff --name-only origin/${GITHUB_PR_TARGET_BRANCH}...HEAD"
+          interpolation: false
+          watch:
+            - path:
+                - .go-version
+                - .buildkite/packaging.pipeline.yml
+                - .buildkite/pipeline.yml
+                - .buildkite/scripts/dra.sh
+                - .buildkite/scripts/packaging/
+                - dev-tools/packaging/
+              config:
+                trigger: "beats-packaging-pipeline"
```

Comment:
could we use the `buildkite-agent pipeline upload` instead as done e.g. in other repos https://github.com/elastic/elastic-agent/blob/main/.buildkite/pipeline.yml#L361C1-L362C1 as we are hoping to eventually switch to this in beats too, to allow us leverage GH comments from the PR bot?

Reply:
I prefer that we don't proceed; there are known issues with the top-level environment variables that I encountered when trying to transition from trigger to `buildkite-agent pipeline upload` in https://github.com/elastic/beats/pull/43161.